developer-docs-site/docs/index.md

```diff
@@ -0,0 +1,142 @@
+---
+title: "Welcome"
+slug: "/"
+hidden: false
+sidebar_position: 0
+hide_table_of_contents: true
+---
+
+# Aptos Developer Documentation
+
+Welcome! At Aptos Labs we are building a Layer 1 for everyone. This documentation will help you develop applications for the Aptos Blockchain, run nodes, and be a part of the blossoming Aptos community. This documentation covers both basic and advanced topics. Here you will find concepts, how-to guides, quickstarts, tutorials, API references, code examples, release notes, and more.
+
+:::tip Aptos Incentivized Testnet-2 (AIT-2)
+**Looking for the AIT-2 docs?** [Click here](nodes/ait/index.md). 
+:::
+
+## Start here
+
+<div class="docs-card-container">
+<div class="row row-cols-1 row-cols-md-2a g-4">
+  <div class="col">
+    <div class="card card-body h-100 d-flex flex-column">
+    <a href="#" class="card-title card-link stretched-link"> <h2>New to Aptos?</h2></a>
+    <p class="card-text">Start with this big picture, Aptos mental model, document.</p>
```

Comment:
nit: This sentence is a bit hard to understand. May be just "Start with this high-level mental model"?